APIMatic.Core/Types/Sdk/CoreRequest.cs

```diff
@@ -103,6 +111,25 @@ public void AddQueryParameters(Dictionary<string, object> queryParamaters)
                 ?? new Dictionary<string, object>(queryParamaters);
         }
 
+        /// <summary>
+        /// Retrieves the value of the "Content-Type" header from the request headers.
+        /// </summary>
+        /// <returns>
+        /// The value of the "Content-Type" header if present; otherwise, <c>null</c>.
+        /// </returns>
+        internal string GetContentType() => Headers?.Where(p => p.Key.EqualsIgnoreCase("content-type"))
+            .Select(x => x.Value)
+            .FirstOrDefault();
+        
+        /// <summary>
+        /// Converts the body of the request to a string representation.
+        /// </summary>
+        /// <returns>
+        /// The string representation of the <see cref="Body"/> if it is not <c>null</c>;
+        /// otherwise, returns an empty string.
+        /// </returns>
+        internal string GetBodyAsString() => Body == null ? String.Empty : Body.ToString();
```

Comment:
use string.Empty instead of String.Empty

Reply:
replaced